Added "synthetic plastic" and "natural plastic"

iwilkie opened the pull request:
Dear Editors,

I have added "synthetic plastic" and "natural plastic", as mentioned in issue #1131.

We will add axioms at a later date.

Thank you!

raissameyer commented on src/envo/envo-edit.owl:
```diff
@@ -40017,6 +40019,28 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <htt
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_06105111> "natural plastic"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_06105111> <http://purl.obolibrary.org/obo/ENVO_06105101>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105112> (synthetic plastic)
```
Following-up on our side-conversation, we could add `manufactured plastic` as a synonym

pbuttigieg replied:
And please mark (e.g. with an editor note) that you'll add the corresponding subclass axiom in the second phase of the project

raissameyer commented on src/envo/envo-edit.owl:
```diff
@@ -40017,6 +40019,28 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <htt
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_06105111> "2021-05-07T09:09:37Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <http://purl.obolibrary.org/obo/ENVO_06105111> "envoPlastics")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_06105111> "natural plastic"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_06105111> <http://purl.obolibrary.org/obo/ENVO_06105101>)
```
Though not directly related to this term, I noticed that its parent class `plastic` has the comment as an annotation property on the definition, instead of on the class level. Perhaps we could correct that and fold it into this PR.

raissameyer reviewed (changes requested):
Thanks for the additions @iwilkie!

Please see below for some comments

pbuttigieg commented on src/envo/envo-edit.owl:
```diff
@@ -40017,6 +40019,28 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <htt
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105111> (natural plastic)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105111> "A plastic which is composed primarily of polymers produced by processes which occur without human intervention."@en)
```
Slight rephrasing:

>A plastic which is composed primarily of polymers produced by processes which occur without human intervention.


To: 

>Plastic which is composed primarily of polymers produced by processes whose agents do not include humans or their technologies.

pbuttigieg reviewed (changes requested):
Slight rephrasing and then good to go

iwilkie commented:
@pbuttigieg and @raissameyer thank you for your comments.

I have moved the comment on "plastic" to the class level, and also added it to the `envoPlastics` subset.

I changed the definition of "natural plastic" as was indicated, added the synonym to "synthetic plastic" as an exact synonym, and added an editor note to both terms indicating that axioms will be added in the second phase.

Please let me know if there are further changes I should make.

pbuttigieg reviewed (approved):
Looks good! Mergable from my side

raissameyer reviewed (approved):
Looks good @iwilkie - I'll merge the PR
